databases/snowflake/batch_prediction_api/snow_batch_prediction_api_query.ipynb

```diff
@@ -0,0 +1,348 @@
+     "start_time": "2020-03-21T08:46:37.267396Z"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "# credential lookup\n",
+    "CREDENTIALS = 'snow_community_credentials'\n",
+    "\n",
+    "response = requests.get(\n",
+    "        DR_APP_HOST + '/api/v2/credentials/',\n",
+    "        headers=DR_MODELING_HEADERS,\n",
+    "    )\n",
+    "\n",
+    "df = pd.io.json.json_normalize(response.json()['data'])[['credentialId', 'name']]\n",
+    "CREDENTIALS_ID = df[df['name'] == CREDENTIALS]['credentialId'].iloc[0]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "ExecuteTime": {
+     "end_time": "2020-03-21T08:40:27.963686Z",
+     "start_time": "2020-03-21T08:39:01.550Z"
+    }
+   },
+   "source": [
+    "### Creating Session and Running Batch Prediction API Job"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "metadata": {
+    "ExecuteTime": {
+     "end_time": "2020-03-21T08:46:40.042602Z",
+     "start_time": "2020-03-21T08:46:40.039457Z"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "session = requests.Session()\n",
+    "session.headers = {\n",
+    "    'Authorization': 'Bearer {}'.format(API_KEY),\n",
+    "}"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "metadata": {
+    "ExecuteTime": {
+     "end_time": "2020-03-21T08:46:40.363864Z",
+     "start_time": "2020-03-21T08:46:40.359538Z"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "job_details = {\n",
+    "    \"deploymentId\": DEPLOYMENT_ID,\n",
+    "    \"numConcurrent\": 4,\n",
+    "    \"passthroughColumns\": [\"PASSENGERID\"],\n",
+    "    \"includeProbabilities\": True,\n",
+    "    \"predictionInstance\" : {\n",
+    "        \"hostName\": DR_PREDICTION_HOST,\n",
+    "        \"datarobotKey\": DATAROBOT_KEY\n",
+    "    },\n",
+    "    \"intakeSettings\": {\n",
+    "        \"type\": \"jdbc\",\n",
+    "        \"fetchSize\": 100000,\n",
+    "        \"dataStoreId\": DATA_CONNECTION_ID,\n",
+    "        \"credentialId\": CREDENTIALS_ID,\n",
+    "        #\"table\": \"PASSENGERS_500K\",\n",
+    "        #\"schema\": \"PUBLIC\",\n",
+    "        \"query\": \"select * from PASSENGERS\"\n",
+    "    },\n",
+    "    'outputSettings': {\n",
+    "        \"type\": \"jdbc\",\n",
+    "        \"table\": \"PASSENGERS_SCORED_BATCH_API\",\n",
+    "        \"schema\": \"PUBLIC\",\n",
+    "        \"statementType\": \"insert\",\n",
+    "        \"dataStoreId\": DATA_CONNECTION_ID, \n",
+    "        \"credentialId\": CREDENTIALS_ID\n",
+    "    }\n",
+    "}"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 8,
+   "metadata": {
+    "ExecuteTime": {
+     "end_time": "2020-03-21T08:46:42.658920Z",
+     "start_time": "2020-03-21T08:46:40.903948Z"
+    }
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "queued batch job: https://app.datarobot.com/api/v2/batchPredictions/1234567891234567890/\n"
```

Comment:
This looks like a scrambled number, right?

Reply:
it's fake